src/opera/test/util/test_h5_utils.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+
+"""
+======================
```

Comment:
Small nit, but the length of the bars here should match the length of the module name, sphinx will complain about this otherwise.

Reply:
Good to know thanks.